docs/hub/model-repos.md

```diff
@@ -180,3 +180,23 @@ SIL Open Font License 1.1	| `ofl-1.1`
 University of Illinois/NCSA Open Source License	| `ncsa`
 The Unlicense	| `unlicense`
 zLib License	| `zlib`
+
+### How can I fork a repository with LFS pointers ?
+
+With "repoA" as the original repository forked in "repoB" as your repository, follow this procedure in order not to break LFS files while rebasing your fork "repoB" with "repoA". If you don't need to rebase, ignore the `git checkout -b temp repoA/main` and `rebase` part, just use the `git lfs *` parts.
```

Comment:
I think something like this may sound a little clearer :) :

```suggestion
When you want to fork a repository with LFS pointers, you need to be careful you don’t break the LFS files when you rebase your fork.

For example, say you have an original repository, **repoA**, and it’s fork, **repoB**. Here is how you can safely fork and rebase **repoB** with **repoA** without breaking anything:
```

Reply:
I would also consider placing this section above the list of licenses, since users will probably be more interested in reading this.